CMakeLists.txt

```diff
@@ -149,7 +149,7 @@ if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
         "include/BetterSMS/objects/*.hxx"
     )
 
-    add_executable(BetterSunshineEngine ${BETTERSMS_SRC})
+    add_executable(BetterSunshineEngine ${BETTERSMS_SRC} "src/patches/modokiTeselafix.cpp")
```

Comment:
You can revert this change because the wild card imports above already handle it.